internal/generator/vector/output/splunk/splunk_sink.toml

```diff
@@ -10,14 +10,33 @@ if err != null {
 	.@timestamp = ts
 }
 
+'''
```

Comment:
Please modify the timestamp logic to use the "timestamp" field in lieu of '@timestamp' or maybe this is a different refactoring so its less changes to the toml files

Reply:
yes, lets do it in separate PR